_sass/_includes.scss

```diff
@@ -13,6 +13,11 @@
 // Get our canvas setup
 @import "vendor/canvas";
 
+$screen-medium: 640px !default;
```

Comment:
Because foundation doesn't expose them simply. So the includes has become like the setup.

Where would you put it?